module13and14/LoopCanvas/LoopCanvas/Views/RenameSongSheet.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  RenameSongSheet.swift
+//  LoopCanvas
+//
+//  Created by Peter Rice on 7/20/24.
+//
+
+import SwiftUI
+
+struct RenameSongSheet: View {
+  @ObservedObject var viewModel: CanvasViewModel
+  @State var songName: String = ""
+  @Binding var showingRenameSongView: Bool
+
+  var body: some View {
+    NavigationView {
+      VStack {
+        Form {
+          Section(header: Text("Thumbnail")) {
+            if let snapshot = viewModel.canvasSnapshot {
+              Image(uiImage: snapshot)
+                .shadow(radius: /*@START_MENU_TOKEN@*/10/*@END_MENU_TOKEN@*/)
```

Comment:
something wierd here?

Reply:
very wierd! these comments are not showing up in the IDE? maybe some weird hidden character? The '10' does look highlighted in blue. Deleting and re-typing seems to remove the highlight.